wdl-format/tests/format/seaseq-case/source.formatted.wdl

```diff
@@ -0,0 +1,939 @@
+    #3. Bowtie INDEX files if not provided
+    String string_spikein = "1"
+    Array[String] string_spikein_buffer = [
+        1,
+    ]
+    if (!defined(spikein_bowtie_index) && defined(spikein_reference)) {
+        # create bowtie index on spikein genome
+        call bowtie.index as spikein_bowtie_idx { input: reference = select_first([
+            spikein_reference,
+            string_spikein,
+        ]) }
+    }
+
+    #4. Make sure indexes are six else build indexes for Spike-in DNA
+    if (defined(spikein_bowtie_index)) {
+        # check total number of bowtie indexes provided
+        Array[File] int_spikein_bowtie_index = select_first([
+            spikein_bowtie_index,
+            string_spikein_buffer,
+        ])
+        if (length(int_spikein_bowtie_index) != 6) {
+            # create bowtie index if 6 index files aren't provided
+            call bowtie.index as spikein_bowtie_idx_2 { input: reference = select_first([
+                spikein_reference,
+                string_spikein,
+            ]) }
+        }
+    }
+    Array[File] actual_spikein_bowtie_index = select_first([
+        spikein_bowtie_idx_2.bowtie_indexes,
+        spikein_bowtie_idx.bowtie_indexes,
+        spikein_bowtie_index,
+        string_spikein_buffer,
+    ])
+
+    # FASTA faidx and chromsizes and effective genome size
+    call samtools.faidx as samtools_faidx {
+        # create FASTA index and chrom sizes files
+        input: reference = reference }
+    call util.effective_genome_size as egs {
+        # effective genome size for FASTA
+        input: reference = reference }
+
+    # Process FASTQs
+    if (defined(sample_fastq)) {
+
+        Array[String] string_fastq = [
+            1,
+        ]  #buffer to allow for fastq optionality
+        Array[File] s_fastq = select_first([
+            sample_fastq,
+            string_fastq,
+        ])
+
+        Array[File] sample_fastqfile = s_fastq
+    }
+    Array[File] original_fastqfiles = flatten(select_all([
+        sample_srafile,
+        sample_fastqfile,
+    ]))
+
+    ### ------------------------------------------------- ###
+    ### ---------------- S E C T I O N 1 ---------------- ###
+    ### ----------- B: remove Spike-IN reads ------------ ###
+    ### ------------------------------------------------- ###
+
+    # if multiple fastqfiles are provided
+    Boolean multi_fastq = if length(original_fastqfiles) > 1 then true else false
+    Boolean one_fastq = if length(original_fastqfiles) == 1 then true else false
+
+    if (defined(spikein_bowtie_index) || defined(spikein_reference)) {
+        scatter (eachfastq in original_fastqfiles) {
+            call fastqc.fastqc as spikein_indv_fastqc { input:
+                inputfile = eachfastq,
+                default_location = if (one_fastq) then sub(basename(eachfastq), ".fastq.gz|.fq.gz", "") + "/SpikeIn/FastQC" else "SAMPLE/" + sub(basename(eachfastq), ".fastq.gz|.fq.gz", "") + "/SpikeIn/FastQC",
```

Comment:
Discussed via Slack, but line length. The `if..then..else` should get line split.